src/main/java/com/contrastsecurity/maven/plugin/InstallAgentContrastMavenMojo.java

```diff
@@ -75,10 +92,10 @@ public String computeAppVersion(Date currentDate) {
     }
 
     public String buildArgLine(String currentArgLine) {
-        return buildArgLine(currentArgLine, appName);
+        return buildArgLine(currentArgLine, appName, serverName);
     }
 
-    public String buildArgLine(String currentArgLine, String applicationName) {
+    public String buildArgLine(String currentArgLine, String applicationName, String serverName) {
```

Comment:
serverName is already a variable in AbstractMojo so passing it to this method masks that with another local variable. Unfortunately we've done the same with applicationName. 

Rename the variables being passed into this method so that they don't conflict/mask variables that already exist.

Reply:
Renamed them to `retrievedApplicationName` and `retrievedServerName` 👍